glean/bin/create_header.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os.path
```

Comment:
This is an interesting one :-D 

The _canonical_ way to do this would be to extend the glean-parser like [FOG does](https://searchfox.org/mozilla-central/source/toolkit/components/glean/build_scripts/glean_parser_ext/cpp.py). However, for the sake of expediency and since this is a very small piece of code, I'm ok with keeping it as it is. Know that this will break when we'll change the metrics.yaml format (we keep evolving that format, and that's why we provide the glean-parser). Maybe you can add a comment to the top, mentioning that if this file gets more complicated or breaks frequently the canonical approach should be re evaluated.